Framework/API/inc/MantidAPI/NearestNeighbourInfo.h

```diff
@@ -48,7 +50,7 @@ class MANTID_API_DLL NearestNeighbourInfo {
 
 private:
   const MatrixWorkspace &m_workspace;
-  Geometry::NearestNeighbours m_nearestNeighbours;
+  std::unique_ptr<NearestNeighbours> m_nearestNeighbours;
```

Comment:
Because we need to build a list of spectrum numbers first, which I could not to in the constructor init list.